molecule/provisioners/dockerprovisioner.py

```diff
@@ -161,7 +161,6 @@ def build_image(self):
                         tag_string))
```

Comment:
Why does the dockerprovisioner not use the state file, and use it's own status.

/cc @kireledan